lib/primer/view_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Provides a convenient shorthand for components.
+module Primer::ViewHelper
+  Components = Hash.new do |hash, component|
+    hash[component] = "Primer::#{component.to_s.camelize}Component".constantize
+  end
+
+  def primer(component, **kwargs, &block)
+    render Components[component].new(**kwargs), &block
+  end
```

Comment:
I'd like to see some error handling here 😄

Reply:
Good thought. The most likely due to the helper is a NameError. Although my default handling would be to re-throw it.